drivers/CoreIMU/source/CoreAccelerometer.cpp

```diff
@@ -0,0 +1,15 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include "CoreAccelerometer.h"
+
+namespace leka {
+
+auto CoreAccelerometer::getAccelXYZ() -> std::tuple<mg_t, mg_t, mg_t>
+{
+	auto data = _core_lsm6dsox.updateData();
+	return std::tuple {data.ui.xl.mg[0], data.ui.xl.mg[1], data.ui.xl.mg[2]};
```

Comment:
In `data.ui.xl.mg` I understand the `data` and the `xl` but the rest is obscure. 

What is `ui`? Why indexes and not `x/y/z`?